ui/src/message_popup/author/questions_page.jsx

```diff
@@ -0,0 +1,162 @@
+      });
+      this.setState({questions});
+    }
+  },
+
+  render(){
+    var archivedQuestions = [];
+    archivedQuestions.push({
+      studentIds: [1, 2, 3],
+      id: 3000,
+      text: "This is an example of a deleted card",
+      examples: [],
+      nonExamples: [],
+      indicatorId: 501,
+      learningObjectiveId: 50
+    });
+    return(
+      <div>
+        <NavigationAppBar
+          title="Message PopUp Questions"
+          iconElementRight={<IconButton onTouchTap={this.onNewQuestion}><AddIcon /></IconButton>}
+          prependMenuItems={
+            <div>
+              <MenuItem
+                onTouchTap={this.onNewQuestion}
+                leftIcon={<AddIcon />}
+                primaryText="New Question" />
+              <MenuItem
+                onTouchTap={this.onArchivedQuestions}
+                leftIcon={<ArchiveIcon />}
+                primaryText="Archived Questions" />
+            </div>
+          }
+          />
+        <div style={styles.container}>
+          <div style={styles.searchbar}>
+            <SearchIcon />
+            <TextField
+              hintText="Search for a specific question"
+              fullWidth={true}
+              style={styles.searchbarText}
+              underlineShow={false}
+              onChange={this.onSearchBarChange}
+            />
+          </div>
+          <Divider />
+          <div style={styles.questionsContainer}>
+            <Paper style={styles.currentQuestionsContainer} rounded={false}>
+              {this.state.questions.map(question => {
+                return (
+                 <QuestionButton question={question} key={question.id} />
+                 );
+              })}
+            </Paper>
+            <Card style={styles.archivedQuestionsContainer} rounded={false} expanded={this.state.showArchivedQuestions} onExpandChange={function(){this.setState({showArchivedQuestions: !this.state.showArchivedQuestions});}.bind(this)}>
+              <CardHeader 
+                title="Archived/Deleted Questions"
+                actAsExpander={true}
+                showExpandableButton={true}/>
+              <CardText expandable={true}>
+                <div>
+                  {archivedQuestions.map(question => <QuestionButton question={question} key={question.id}/>)}
+                </div>
+              </CardText>
+            </Card>
+          </div>
+        </div>
+      </div>
+      );
+  }
+});
+
+const styles = {
+  container: {
+    padding: 0
+  },
+  searchbar: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  searchbarText: {
+    paddingLeft: 10,
+    paddingRight: 5
+  },
+  questionsContainer: {
+    position: 'absolute',
+    width: '100%',
+    top: 112,
+    bottom: 0,
+    overflowY: 'scroll'
+  },
+  currentQuestionsContainer: {
+    marginBottom: 10
+  },
+  archivedQuestionsContainer: {
+    marginBottom: 10,
+    backgroundColor: "#F4F4F4"
```

Comment:
Can you use a [Material UI color](http://www.material-ui.com/#/customization/colors) here?